### PR TITLE
Supports AWS Code Commit.

### PR DESCRIPTION
**Pull request by crossroad0201**
Added support for the [AWS Code Commit](https://aws.amazon.com/codecommit/?nc1=h_ls).

Thank you.

### CODE REVIEW

**Comment by derimagia** [comment]
Thanks!

Would appreciate it if you're able to write tests for this. Otherwise we'll get around to it

**Comment by crossroad0201** [comment]
OK!
I will add a test cases :thumbsup:

**Comment by crossroad0201** [comment]
@derimagia 
I added tests.
Please check that.

File: git-open [hunk 1]
```diff
@@ -198,6 +198,11 @@ elif [[ "${#pathargs[@]}" -ge '2' && ${pathargs[${#pathargs[@]} - 2]} == '_git'
   fi
 elif [[ "$domain" =~ amazonaws\.com$ ]]; then
   # AWS Code Commit
+  if (( is_issue )); then
+      echo "Issue feature does not supported on AWS Code Commit." 1>&2
+    exit 1
```
Comment (derimagia) [inline git-open]: Can you fix the Indentation here?

Comment (crossroad0201) [reply git-open]: Sorry... 
I fixed it.

**Pull request merged**